modules/api/variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "serverless_lambda_api_invoke_arn" {}
```

Comment:
best practice:
variables block should: 
- contain a brief description: to what serve this variable
- type: string, bool, map(string), object, etc
- optional: default value if needed (can be used to not block the execution if the variable if optional, you add default empty)